Source/Atomic/Network/Network.cpp

```diff
@@ -250,9 +287,12 @@ bool Network::StartServer(unsigned short port)
 
     PROFILE(StartServer);
 
+    serverPort_ = port;
+
     if (network_->StartServer(port, kNet::SocketOverUDP, this, true) != 0)
     {
         LOGINFO("Started server on port " + String(port));
+

```

Comment:
There is a fair amount of log, maybe go through and remove what was generated while debugging and change the ones to keep to LOGDEBUG variants?